src/components/hide-this-page/index.md.njk

```diff
@@ -0,0 +1,98 @@
+---
+title: Hide this page
+description: Use the button component to help users carry out an action
+section: Components
+aliases:
+backlog_issue_id: 213
+status: Experimental
+layout: layout-pane.njk
+---
+
+This component is currently experimental because more research is needed to validate it.
+
+{% from "_example.njk" import example %}
+{{ example({group: "components", item: "hide-this-page", example: "default", html: true, nunjucks: false, open: false}) }}
+
+## When to use this component
+
+The ‘Hide this page’ component helps users to exit a service, website or application quickly and safely.
+
+Use the component when use of a service could put people in danger or at risk of harm. For example, a service about domestic abuse being used by a potential victim.
+
+## When to not use this component
+
+Do not use this component if the service or part of the service is unlikely to put a user at risk.
+
+## How it works
+
+There are 2 ways to use the warning text component. You can use HTML or, if you are using Nunjucks or the GOV.UK Prototype Kit, you can use the Nunjucks macro.
+
+### Help users cover their tracks
+
+Using the component, should help obscure any evidence that a user has visited the service or website.
+
+When a user opts to ‘Hide this page’, this should:
+
+ - change the current tab to a frequently visited website, for example Google obscure the previous page history
+ - open a new tab as the active tab. This should be a frequently visited website, for example the BBC Weather website
+
+### Use in every place where users are at risk
+
+Add the ‘Hide this page’ component wherever a user could be at risk. This could cover a whole service or parts of a journey where the information is sensitive and could alert a perpetrator.
+
+### Make the component clearly visible
+
+Add the ‘Hide this page’ component to the top of every screen you identify as having the potential to put a user at risk.
+
+The component is made up of two parts:
+
+**1. Visible button**
+
+This should be placed just beneath the header or phase banner.
+
+The button component should always be clearly visible and easy to reach so a user can hide the page immediately when needed.
+
+Use the recommended colour to highlight the importance of the button and make it easy to locate in an emergency situation.
+
+The component can be enabled/disabled by adding or removing the class ```govuk-quick-exit__enabled```.
+
+**2. Screen reader and spacing for mobile devices**
+
+This should be at the very top of the page, above the skip to main content link.
```

Comment:
I like this - working on the COVID services we struggled with where the component should go in the markup and this seems such a clear obvious solution 👍